test/test_effects.rb

```diff
@@ -7,12 +7,12 @@
 class TestEffects < Minitest::Test
   extend RDL::Annotate
 
-  type :pure_method, '(Integer) -> Integer [.]', typecheck: :effects
-  def pure_method(x)
-    x
-  end
+  # type :pure_method, '(Integer) -> Integer [.]', typecheck: :effects
```

Comment:
Uncomment this test

Reply:
2c975cf07cadfb7fa87e4e81f2a45803d02d9222
Uncommented the test.